collie/cross_validation.py

```diff
@@ -282,10 +295,19 @@ def _stratified_split(interactions: BaseInteractions,
     return train_interactions, test_interactions
 
 
-def _stratified_split_parallel_worker(idxs_to_split: Iterable[Any],
+def _stratified_split_parallel_worker(interactions: BaseInteractions,
+                                      idxs_to_split: Iterable[Any],
                                       test_p: float,
                                       seed: int,
                                       force_split: bool) -> np.array:
+    try:
+        if interactions.negative_sample_type == 'item':
+            opposite_type = 'user'
+        elif interactions.negative_sample_type == 'user':
+            opposite_type = 'item'
+    except AttributeError:
+        opposite_type = 'user'
```

Comment:
Could we try just passing in `opposite_type` to each worker in the parent function above so we don't have to re-compute this each time.